server/utils/formUtils.ts

```diff
@@ -125,3 +126,21 @@ export const escape = (text: string): string => {
   const escapeFilter = new nunjucks.Environment().getFilter('escape')
   return escapeFilter(text).val
 }
+
+export const generateResponsesForYesNoAndCommentsSections = (
```

Comment:
I wonder if this belongs in `form-pages/utils/index.ts`?

Reply:
Yeah that makes sense